### PR TITLE
feat:[CUS-170] Added ec2 launch template.

### PR DESCRIPTION
**Pull request by kushZemoso**
# Description

Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List
any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Chore (no code changes)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also
list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] My commit message/PR follows the contribution guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

# **Other information**:

List any documentation updates that are needed for the Wiki

### CODE REVIEW

**Review by arunpaladin — APPROVED** [review]
Good.
Can u please updated the unit test for modified polices.

File: jobs/pacman-cloud-discovery/src/main/java/com/tmobile/cso/pacman/inventory/util/InventoryUtil.java [hunk 1]
```diff
@@ -3172,6 +3104,52 @@ public static Map<String, List<DescribeHubResult>> fetchSecurityHub(BasicSession
...
+						}
+					});
+
+					if (!launchTemplateVHList.isEmpty()) {
+						log.debug(InventoryConstants.ACCOUNT + accountId + " Type: Launch Template " + region.getName() + " >> " + launchTemplateVHList.size());
+						launchTemplates.put(accountId + delimiter + accountName + delimiter + region.getName(), launchTemplateVHList);
+					}
+				}
+			}catch(Exception e){
+				log.warn(expPrefix+ region.getName()+InventoryConstants.ERROR_CAUSE +e.getMessage()+"\"}");
```
Comment (raviteja-mandala) [inline jobs/pacman-cloud-discovery/src/main/java/com/tmobile/cso/pacman/inventory/util/InventoryUtil.java:3147]: it should be log.error

Comment (kushZemoso) [reply jobs/pacman-cloud-discovery/src/main/java/com/tmobile/cso/pacman/inventory/util/InventoryUtil.java:3147]: For every collector in aws we are use log.warn only

**Comment by kushZemoso** [comment]
> Good. Can u please updated the unit test for modified polices.

D

**Review by santhosh-challa — APPROVED** [review]

**Pull request merged**